public/app/percona/backup/components/AddBackupPage/AddBackupPage.tsx

```diff
@@ -110,6 +112,16 @@ const AddBackupPage: FC<GrafanaRouteComponentProps<{ type: string; id: string }>
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const onDemandChange = useCallback(() => {
+    setQueryParams({ scheduled: null });
+    setModalTitle(Messages.getModalTitle(false, editing));
+  }, [editing, setQueryParams]);
+
+  const onSheduledChange = useCallback(() => {
```

Comment:
Should be "scheduled"

Reply:
Are you sure? 
![image](https://user-images.githubusercontent.com/90199600/201641613-1a2e57e6-4361-49be-b861-b6f2187458eb.png)
I checked how it was in previous version and there was 2 times "editing"

Reply:
I mean there's a typo: "onScheduledChange" and not "onSheduledChange" 😅

Reply:
![image](https://user-images.githubusercontent.com/90199600/201642020-14ae4f6e-52fa-4196-b967-a324bbd03456.png)

Reply:
or) yes) sorry! I see now

Reply:
fixed